### DISCUSSION TITLE
Browser support?

### DISCUSSION THREAD

**Discussion by jacwright** [jacwright/typewriter#58] [General]
Typewriter only supports evergreen browsers right now. No IE. Is that something we can live with? I can. 😀 

It would need an audit to make it work. The things I can think of now are Array forEach/map, etc and the use of the template element in the view `setHTML()` method.

**Comment by scarroll32** [comment]
I think that's fine, will add something to the README.

**Comment by tomcon** [comment]
> Is that something we can live with? I can. 

Me too :) 

Am defaulting currently to quill on a large migration but hope to get involved here if possible when things settle. Am very interested to see how the project develops.

Good work!
